index.bs

```diff
@@ -931,6 +933,50 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Advertise Platform Authenticator Support - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method can be used by [=[RPS]=] to inform the user that they can create a new credential with one or more platform authenticators.
+The method gives [=[RPS]=] the ability to inform the client that they support registering credentials with [=platform authenticators=]
+while giving users control over whether this information is appropriate to provide to the RP.
+When the method is invoked, the [=client=] will search for the existence of a [=platform authenticator=]
+capable of registering credentials.
+Upon finding one, the [=client=] will determine whether the user would be interested in
+creating a credential based on a number of factors, which may include the user's explicitly stated intentions.
+If such interest is found, the promise will be resolved with the value of <i>True</i>.
+Based on the result, the [=[RP]=] can take further actions to guide the user to create a credential.
+
+This method has no arguments and returns a boolean value.
+
+<pre class="idl">
+    [SecureContext]
+    partial interface PublicKeyCredential {
+        [Unscopable] Promise < boolean > isPlatformAuthenticatorReady();
+    };
+</pre>
+
+When this method is invoked, the user agent MUST execute the following algorithm:
+
+1. Let |promise| be a [=a new Promise=]. Return |promise| and execute the following steps [=in parallel=].
+
+1. Using a platform-specific procedure, enumerate all the [=platform authenticators=] on this platform.
+    If there are no such authenticators, resolve |promise| with a value of <i>False</i> and terminate this algorithm.
+
+1. If one or more [=platform authenticators=] were found, determine the user's willingness to create a credential
+    for this [RP] on such an authenticator. Factors used in this determination may include, for example:
+
+    1. Whether the user is running in private or incognito mode.
+    1. Whether the user has configured the [=client=] to never create such credentials.
+    1. Whether the user has previously expressed an unwillingness to create a new credential for this [=[RP]=],
+        either through configuration or by declining a user interface prompt.
+    1. The user's explicitly stated intentions, as obtained through user interaction with the user agent.
+
```

Comment:
suggest compressing the above (lines 963 to 972) down to..
>Otherwise, assess whether the user is willing to create a credential for this [RP] using one of the available  [=platform authenticators=] (see above for a discussion of the factors that may be employed by the [=client=] in making this assessment).

note: further above, i suggest moving the example assessment criteria up to the alg introduction text.